ts/features/itwallet/machine/eid/actors.ts

```diff
@@ -151,6 +152,10 @@ export const createEidIssuanceActorsImplementation = (
     }
     assert(sessionToken, "sessionToken is undefined");
 
-    await revokeCurrentWalletInstance(sessionToken, integrityKeyTag.value);
+    try {
+      await revokeCurrentWalletInstance(sessionToken, integrityKeyTag.value);
+    } catch (e) {
+      itwSendExceptionToSentry(e, "revokeWalletInstance");
+    }
```

Comment:
Can we remove the try/catch to the revokeCurrentWalletInstance function?
Don't forget to retrow the error, or the machine will never know that if failed